src/core/foot/foot.view.js

```diff
@@ -39,15 +39,19 @@ export default class FootView extends View {
 
 	value(column) {
 		if (column.aggregation) {
-			if (!Aggregation.hasOwnProperty(column.aggregation)) {
+			let aggregation = column.aggregation,
```

Comment:
should be const,
each line should contains modifier e,g,
const aggregation = ;
const aggregationOptions = ;